src/getThreadList.js

```diff
@@ -52,11 +52,11 @@ module.exports = function(defaultFuncs, api, ctx) {
         if (resData.error) {
           throw resData;
         }
-        log.verbose("Response in getThreadList: " + JSON.stringify(resData.payload.threads));
+        log.verbose("getThreadList", JSON.stringify(resData.payload.threads));
```

Comment:
We could remove this `JSON.stringify` here as we're not concatenating two strings anymore (I'm sure the logger supports printing objs).